src/components/ModalExport.tsx

```diff
@@ -210,9 +209,46 @@ const ModalExport = () => {
     setPublishing(false)
   }
 
+  const [advancedSettings, setAdvancedSettings] = useState(false)
+
+  /** Toggles advanced setting when Advanced CTA is clicked. */
+  const onAdvancedClick = () => setAdvancedSettings(!advancedSettings)
+
+  const [shouldIncludeMetaAttributes, setShouldIncludeMetaAttributes] = useState(true)
+
+  /** Updates checkbox value when clicked and set the appropriate value in the selected option. */
+  const onChangeLosslessCheckbox = () => {
+    const updatedValue = {
+      ...selected,
+      excludeMeta: !shouldIncludeMetaAttributes
+    }
+    setSelected(updatedValue)
```

Comment:
The `ExportOption` type needs to be refactored since we are handling `excludeMeta` as an independent variable now. `ExportOption` should no longer contain `excludeMeta`. We're halfway in between right now which is resulting in some extraneous structure for what should be a single, local component state (`shouldIncludeMetaAttributes`).

The fact that you have to call `setSelected` and `setShouldIncludeMetaAttributes` to update a single value is a sign that they should be combined.

Reply:
Changed. Also I created `ChevronImg` as it was mostly same code duplicated.